SpatialGDKEditorCommandlet/Source/Private/Commandlets/SchemaCommandlet.cpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "SchemaCommandlet.h"
+#include "SpatialGDKEditorCommandletPluginPrivate.h"
+#include "SpatialGDKEditor.h"
+
+USchemaCommandlet::USchemaCommandlet()

```

Comment:
Could an initializer list be used here?

Reply:
No, initializer list is only good for members of the class (not members of the base class, which is what these are)